src/Form/AddressBookType.php

```diff
@@ -107,11 +109,11 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     'mapped' => false,
                 ])
                 ->add('telephone', TextType::class, [
-                    'required' => true,
+                    'required' => $this->authorizationChecker->isGranted('ROLE_ADMIN') || $this->authorizationChecker->isGranted('ROLE_DISPATCHER') ? false : true,
```

Comment:
ROLE_ADMIN includes ROLE_DISPATCHER, so I think

```
'required' => $this->authorizationChecker->isGranted('ROLE_DISPATCHER') ? false : true,
```

should be enough

Reply:
you right